.travis.yml

```diff
@@ -35,10 +33,11 @@ before_install:
   - composer self-update
   - if [ "$DEPS" = "dev" ]; then perl -pi -e 's/^}$/,"minimum-stability":"dev"}/' composer.json; fi
   - if [ "$SYMFONY_VERSION" != "" ]; then composer require symfony/symfony:${SYMFONY_VERSION} --no-update; fi
+  - phpunit --version
 
 install: composer update --prefer-dist $COMPOSER_FLAGS
 
-script: vendor/bin/simple-phpunit
```

Comment:
ah, thats how we could control that, thanks.

but it seems that in versions  5.6 that we get, the yaml issue is fixed. or is that a problem that could come up again in the future?